clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/IndexWebService.scala

```diff
@@ -22,19 +25,32 @@ abstract class IndexWebService[CI <: ClioIndex](
   private[webservice] def pathPrefixKey: Directive1[indexService.clioIndex.KeyType]
 
   private[webservice] val postMetadata: Route = {
-    pathPrefix("metadata") {
-      pathPrefixKey { key =>
-        post {
-          entity(as[indexService.clioIndex.MetadataType]) { metadata =>
-            complete(indexService.upsertMetadata(key, metadata))
+    pathPrefix(metadataString) {
+      parameter(Symbol(forceString).as[Boolean] ? false) { force =>
+        pathPrefixKey { key =>
+          post {
+            entity(as[indexService.clioIndex.MetadataType]) { metadata =>
```

Comment:
I don't really understand what is happening in this block or why

Reply:
dan explained it

Reply:
(Probably worth adding a comment about why we force-run the stream here to get a strict `Future` instead of letting `complete` auto-serialize the stream to a chunked entity)